verifier/src/lib.rs

```diff
@@ -91,7 +91,7 @@ pub trait ResultMessage {
 }
 
 /// Failed to verify.
-#[derive(Debug, Eq, PartialEq, Clone)]
+#[derive(Debug, Clone)]
```

Comment:
These got removed due to `MrSignerVerificationMetadata` containing a `VerificationResult<VerifictationResultMetadata>`
The `VerificationResult` can't derive `Eq` and `PartialEq` because it's `Choice` member doesn't.
We currently don't have a need for `Eq` or `PartialEq` so removed them.